### ISSUE TITLE
[Question] Incorrect documentation for the action space

### ISSUE THREAD

**Issue by Martin36** [Farama-Foundation/Minigrid#462]
### Question

In the documentation is says that only the actions 0-3 are used (https://minigrid.farama.org/environments/babyai/Synth/#action-space). However, looking at the [code](https://github.com/Farama-Foundation/Minigrid/blob/master/minigrid/minigrid_env.py#L569) it seems that both the *drop* and *toggle* actions are used.

**Comment by pseudo-rnd-thoughts** [comment]
Yes, but my suspicion is that there is no object to drop so it is "unused"
If I'm wrong, could you make an example script where a drop or toggle actions are necessary?

**Comment by Martin36** [comment]
To be able to solve the goal “put the {color} {type} next to the {color} {type}”, you would need to apply the drop action at least once. 

Toggle is needed whenever the agent needs to navigate to another room, where the door to that room is closed. At least this action would be needed to solve many of the environments.
